ami/main/tests.py

```
@@ -2927,6 +2927,59 @@ def test_taxa_include_occurrence_determinations_not_directly_linked(self):
+        self.client.force_authenticate(user=self.user)
+
+    def _list_occurrences(self, limit: int) -> int:
+        from django.core.cache import caches
+        from django.test.utils import CaptureQueriesContext
+
+        url = f"/api/v2/occurrences/?project_id={self.project.pk}&limit={limit}"
+        # Clear cachalot cache between runs so query counts reflect cold state
+        try:
+            caches["default"].clear()
+        except Exception:
+            pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)